packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -34,7 +34,7 @@ module.exports = (resolve, rootDir, isEjecting) => {
     setupFilesAfterEnv: setupTestsFile ? [setupTestsFile] : [],
     testMatch: [
       '<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}',
-      '<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}',
+      '<rootDir>/src/**/*(*.)@(spec|test).{js,jsx,ts,tsx}',
```

Comment:
I know this regex has gone through a few revisions lately. Are we sure this is correct?

Reply:
This is probably just a mistake since this branch has been hanging around for quite a while. I'll double check but most likely can revert this change.